README.md

```diff
@@ -28,7 +28,7 @@ jobs:
           title: Karma Test Coverage
 ```
 
-## Usage with Jest
+## How to use with Jest
 1. Add `"codeCoverage": true,` under test > options in angular.json
```

Comment:
You are still mentioning angular here, maybe adding to this title also "How to use with Jest + angular"